File: src/main/java/com/gmail/borlandlp/minigamesdtools/activepoints/reaction/custom/TeamIncrementWinTicketsReaction.java
package com.gmail.borlandlp.minigamesdtools.activepoints.reaction.custom;

import com.gmail.borlandlp.minigamesdtools.Debug;
import com.gmail.borlandlp.minigamesdtools.activepoints.reaction.Reaction;
import org.bukkit.entity.Entity;

public class TeamIncrementWinTicketsReaction extends Reaction {
    private int value;

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

    @Override
    public void performDamage(Entity reactionInitiator, double damage) {
        Debug.print(Debug.LEVEL.NOTICE, "TeamIncrementWinTicketsReaction->performDamage(" + reactionInitiator.getName() + ", " + damage + ")");
    }

    @Override
    public void performIntersection(Entity reactionInitiator) {
        Debug.print(Debug.LEVEL.NOTICE, "TeamIncrementWinTicketsReaction->performIntersection(" + reactionInitiator.getName() + ")");
    }

    @Override
    public void performInteraction(Entity reactionInitiator) {

    }

    @Override
    public String toString() {
        return "{Reaction type=TeamIncrementWinTicketsReaction}";
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/activepoints/reaction/custom/TeamIncrementWinTicketsReaction.kt
package com.gmail.borlandlp.minigamesdtools.activepoints.reaction.custom

import com.gmail.borlandlp.minigamesdtools.Debug
import com.gmail.borlandlp.minigamesdtools.Debug.print
import com.gmail.borlandlp.minigamesdtools.activepoints.reaction.Reaction
import org.bukkit.entity.Entity

class TeamIncrementWinTicketsReaction : Reaction() {
    var value = 0

    override fun performDamage(reactionInitiator: Entity, damage: Double) {
        print(
            Debug.LEVEL.NOTICE,
            "TeamIncrementWinTicketsReaction->performDamage(" + reactionInitiator.name + ", " + damage + ")"
        )
    }

    override fun performIntersection(reactionInitiator: Entity) {
        print(
            Debug.LEVEL.NOTICE,
            "TeamIncrementWinTicketsReaction->performIntersection(" + reactionInitiator.name + ")"
        )
    }

    override fun performInteraction(reactionInitiator: Entity) {}
    override fun toString(): String {
        return "{Reaction type=TeamIncrementWinTicketsReaction}"
    }
}